src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
+    }
```

Comment:
do we use this constructor?

Reply:
We should have an empty default constructor https://cln.sh/4JyZTD

Reply:
Agree, my mistake